src/app/modules/main/profile_section/devices/model.nim

```diff
@@ -80,18 +80,13 @@ QtObject:
       of ModelRole.IsCurrentDevice:
         result = newQVariant(item.isCurrentDevice)
 
-  proc addItems*(self: Model, items: seq[Item]) =
+  proc setItems*(self: Model, items: seq[Item]) =
     if(items.len == 0):
```

Comment:
if we already have items this condition may be wrong

Reply:
Nice catch! It's just a leftover from the old function.
Thanks!